### PR TITLE
FWI-3280 - Vv/add docs for reports cfg api/fwi 3280

### PR DESCRIPTION
**Pull request by vitorvezani**
This PR fixes #

## Checklist
* [X] I have signed the CLA
* [X] I have updated/added any relevant documentation

### What's the goal of this PR?
Adds documentation for: 
- Docker registry API - Scan private images
- Auto-scan Reports config.

### CODE REVIEW

File: docs/configure/ci/autoscan.md [hunk 1]
```diff
@@ -8,7 +8,44 @@ meta:
 *Documentation coming soon!*
 
 ## Using Auto-Scan to Scan Private Images
-Scanning private container images is not yet supported in Auto-Scan but is currently on-roadmap.
+You can now scan private container images by setting up docker registries information in Fairwinds Insights via API.
+These information will be used to authenticate/authorize against your docker registry and fetch private image container to be scanned.
```
Comment (lizcoolman) [inline docs/configure/ci/autoscan.md:12]:
```suggestion
This information will be used to authenticate/authorize against your docker registry and fetch the private image container to be scanned.
```

File: docs/configure/ci/autoscan.md [hunk 2]
```diff
@@ -20,10 +57,35 @@ To re-run an Auto-Scan:
 
 Re-running an Auto-Scan job usually takes a few minutes. Look for the 'Completed' status to see your latest results.
 
-## Customizing Auto-Scan Using Fairwinds-insights.yaml
-Unlike the Insights CI integration, Auto-Scan does not require users to create a `fairwinds-insights.yaml` configuration file at the base of their GitHub repository. This is because Auto-Scan will automatically crawl and discover YAML manifests, Helm charts, and docker images available for scanning.
+## Manifests auto-discovery
+Unlike Insights CI integration, Auto-Scan does not require users to create a `fairwinds-insights.yaml` configuration file at the base of their GitHub repository. This is because Auto-Scan will automatically crawl and discover YAML manifests, Helm charts, and docker images available for scanning.
+
+### Reports Configuration
+When using auto-discovery, all reports are `enabled` by default, if you want to further customize. It is possible to manually configure it via API using CURL:
```
Comment (lizcoolman) [inline docs/configure/ci/autoscan.md:64]: When using auto-discovery, all reports are `enabled` by default. To customize this, you can manually configure via API using CURL:

**Review by lizcoolman — COMMENTED** [review]
See updates in comments on the following lines: 
line 12
line 64

**Comment by vitorvezani** [comment]
Fixed Liz, thank you!

**Review by mggude — APPROVED** [review]

**Comment by vitorvezani** [comment]
@mggude should we wait for CI plugin bump (from 2.4 to 4.1) to merge the docs?

**Comment by vitorvezani** [comment]
@mggude added CI plugin version override docs [here](https://github.com/FairwindsOps/insights-docs/pull/266/files#diff-280921bdfd0a6df4ba4ce3ebf72e86a1bbbc7a9e6940c320aec3abeeace467ceR87)

**Pull request merged**